Resources/config/security.xml

```diff
@@ -5,11 +5,12 @@
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
+        <defaults public="false" />
```

Comment:
This one should be removed as it breaks compat with older versions of Symfony.

Reply:
ok ( i dont know why, i didnt received a notification for your review)